README.md

```diff
@@ -22,6 +22,9 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 
 <!-- TOC -->
 
+- [Awesome Docker ![Awesome](https://github.com/sindresorhus/awesome) ![Join the chat at https://gitter.im/veggiemonk/awesome-docker](https://gitter.im/veggiemonk/awesome-docker) ![Say Thanks](https://saythanks.io/to/veggiemonk)](#awesome-docker-awesomehttpsgithubcomsindresorhusawesome-join-the-chat-at-httpsgitterimveggiemonkawesome-dockerhttpsgitterimveggiemonkawesome-docker-say-thankshttpssaythanksiotoveggiemonk)
```

Comment:
??

Reply:
It should have only been a few line changes, so something have happened. I'm gonna check it out.

Reply:
Reverted the changes and fixed the alphabetical order again. Something happened with the table of content section when hitting save, but discarded these changes.